docs/doxygen/timeouts.dox

```diff
@@ -91,4 +98,17 @@ That means, especially in multi-threaded application, the process loop timeout c
 faster than the MQTT Keep Alive interval. If a dummy @ref MQTTGetCurrentTimeFunc_t was passed to @ref mqtt_init_function,
 then the timeout must be set to 0.
 
+@see @ref mqtt_receivetimeout
+
+@section mqtt_timeouts_connect Timeout for MQTT_Connect
+This timeout is passed as an argument to @ref mqtt_connect_function. It is the maximum time to wait for an MQTT CONNACK packet. If this value is
+set to 0, then instead of a time-based loop, it will attempt to call the Transport Receive function up to a fixed maximum number of times,
+which is defined by @ref MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT.
+
+We recommend to choose a timeout for  @ref mqtt_connect_function by experimenting with an MQTT application on a sample of devices and
+obtaining the data for latency of each CONNACK packet from the broker after an MQTT CONNECT packet is sent. A timeout value can then be chosen
+based on the statistical measure suitable for the end application, such as 99th percentile or average.
+
+@see @ref mqtt_receivetimeout
```

Comment:
Added accidentally?

Reply:
No this is a section in which the timeouts are mentioned in short as part of the design